package.json

```diff
@@ -12,7 +12,7 @@
     "format": "prettier --write \"packages/**/*.{ts,js,tsx,jsx,json}\"",
     "lint": "eslint packages/ --ext .js,.ts,.jsx,.tsx",
     "bootstrap": "lerna bootstrap",
-    "publish": "lerna version minor --yes && lerna publish from-git --yes"
+    "release": "lerna version minor --yes && lerna publish from-git --yes"
```

Comment:
why do u rename it?

Reply:
These changes were already on master. I just had to rebase. However, we had to change it since lerna tryed to release it twice